src/qibo/noise.py

```diff
@@ -0,0 +1,81 @@
+from qibo import gates
+
+class PauliError():
+    """Quantum error associated with the `qibo.core.gates.PauliNoiseChannel`.
+
+        Args:
+            options (tuple): see :class:`qibo.core.gates.PauliNoiseChannel`
+    """
```

Comment:
I would use
```suggestion
    """Quantum error associated with the :class:`qibo.abstractions.gates.PauliNoiseChannel`.

        Args:
            options (tuple): see :class:`qibo.abstractions.gates.PauliNoiseChannel`
    """
```
so that the link to the gate docs works properly when viewing the html version that will be online.